components/lib/crash/src/main/java/mozilla/components/lib/crash/prompt/CrashReporterActivity.kt

```diff
@@ -53,7 +54,17 @@ class CrashReporterActivity : AppCompatActivity() {
         val appName = crashReporter.promptConfiguration.appName
         val organizationName = crashReporter.promptConfiguration.organizationName
 
-        binding.titleView.text = getString(R.string.mozac_lib_crash_dialog_title, appName)
+        val isBackground = (crash as? Crash.NativeCodeCrash)?.processType ==
+            Crash.NativeCodeCrash.PROCESS_TYPE_BACKGROUND_CHILD
+        val title = when (isBackground) {
```

Comment:
nit: since we're not using `val title` you can replace this with `binding.titleView.text =`